config.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"bytes"
+	"io/ioutil"
+
+	"go.uber.org/config"
+)
+
+//Config is a struct that will be used to parse the enviroment variables located in config.yml
+type Config struct {
+	Goenv     string
+	Directory string
+	Exclude   string
+}
+
+//CfgFactory generates config based on config.yml file
+func CfgFactory() Config {
+	yamlFile, err := ioutil.ReadFile("config.yml")
+	if err != nil {
+		panic(err)
```

Comment:
Melhor usar log.Fatal
https://stackoverflow.com/questions/35996966/what-are-the-differences-in-outcome-for-panic-vs-log-fatalln-in-golang

Reply:
Nas linhas de baixo também